### DISCUSSION TITLE
rawkeyboard

### DISCUSSION THREAD

**Discussion by pesian** [jlesage/docker-firefox#225] [Q&A] [answered]
"I need to enable "RawKeyboard=1" input in Xvnc, but I can't find any configuration file to activate this feature."

**Comment by jlesage** [comment, marked as answer]
There is currently no facility to set custom xvnc parameters.
I guess you can add the option yourself by manually editting `/etc/services.d/xvnc/params` inside the container.

**Reply by pesian** [reply]
"Thank you, I tried this and it works: "RUN echo 'RawKeyboard=1'  >> /etc/services.d/xvnc/params"
